src/index.html

```diff
@@ -1,22 +1,420 @@
+            <p class="about-us__text">
+              Bang & Olufsen is expanding its Recreated Classics Program with
+              the launch of Beosystems – a limited edition music system that
+              transcends time by bridging the gap between one of our iconic
+              designs from 1972 and today’s cutting-edge digital technology.
+            </p>
+          </section>
+
+          <section
+            class="contact-us"
+            id="contact-us"
+          >
+            <h2 class="section-title">Contact Us</h2>
+            <div class="contact-us__content">
+              <form
+                action="#"
+                method="post"
+                class="contact-us__form"
+                onsubmit="this.reset(); return false;"
+              >
+                <input
+                  class="contact-us__input"
+                  type="text"
+                  name="name"
+                  placeholder="Name"
+                  pattern="\w+"
+                  maxlength="30"
+                  required
+                />
+
+                <input
+                  class="contact-us__input"
+                  type="email"
+                  name="email"
+                  placeholder="Email"
+                  maxlength="50"
+                  required
+                />
+
+                <textarea
+                  type="text"
+                  class="contact-us__input text-area"
+                  placeholder="Message"
+                  required
+                ></textarea>
+
+                <input
+                  type="submit"
+                  class="primary-button contact-us__button"
+                  value="Send"
+                >
+              </form>
+              <ul class="contact-us__contacts-list">
+                <li class="contacts-us__contact-item">
+                  <a href="tel:+1 234 5555-55-55"><span>Phone</span>+123455555555</a>
+                </li>
+                <li class="contacts-us__contact-item">
+                  <a href="mailto:hello@nothing.com"><span>Email</span>hello@nothing.com</a>
+                </li>
+                <li class="contacts-us__contact-item">
+                  <a
+                    href="https://maps.app.goo.gl/ALkpSbNjjR3zfZo57"
+                    target="_blank"
+                  >
+                    <span>Address</span>
+                    400 first ave.
+                    <br />
+                    suite 700Minneapolis,
+                    <br />
+                    MN 55401
+                  </a>
+                </li>
+              </ul>
+            </div>
+          </section>
+
+        </div>
+      </div>
+    </main>
+
+    <footer class="footer">
+      <a
+        href="https://github.com/TargoniyAnatoliy"
+        target="_blank"
+      >
+        Created by Ma-Student ©
+      </a>
+    </footer>
+    <a
+      href="#page-top"
+      class="on-top"
+    >
+      &uarr;
+    </a>
   </body>
```

Comment:
Add empty lines between multiline sibling blocks of HTML

![image](https://github.com/user-attachments/assets/7b31cade-615c-4e16-9db5-e9c611916adc)